src/tools/interop/idt/.gitignore

```diff
@@ -8,3 +8,4 @@ pycache/
 venv/
 .zip
 BUILD
+urls.txt
```

Comment:
maybe `prober_urls` would be easier to maintain given that we seem to be able to give any name to this file

Reply:
Thanks I've changed it to exactly that!